man/kernel-install.xml

```diff
@@ -255,6 +255,11 @@
       <filename>install.conf</filename>, <filename>entry-token</filename>, and other files will be
       read from this directory.</para>
 
+      <para><varname>$KERNEL_INSTALL_PLUGINS</varname> can be set to override the list of plugins executed by
+      <command>kernel-install</command>. The argument is a newline-separated list of paths.
+      <literal>KERNEL_INSTALL_PLUGINS=:</literal> may be used to prevent any plugins from running.
```

Comment:
hmm, newline separated? that's quite unusual. Sounds like unnecessarily hard to use? why not just use ":" or some more commonly used seperator in shell scripts?

It's not that we really should care about people who are crazy enough to encode ":" in their plugin name.

Reply:
Updated with "whitespace-seperated list".